src/Documentation/tutorials_and_samples/tutorial_1_helloworld.md

```diff
@@ -0,0 +1,151 @@
+
+After loading the configurations, the SiloHost is built and then started asynchronously.
+
+## Configuring the client
+
+Similar to the silo, the client is configured via `ClientBuilder` and a similar collection of option classes.
+
+```csharp
+        private static async Task<IClusterClient> StartClientWithRetries()
+        {
+            attempt = 0;
+            IClusterClient client;
+            client = new ClientBuilder()
+                .UseLocalhostClustering()
+                .Configure<ClusterOptions>(options =>
+                {
+                    options.ClusterId = "dev";
+                    options.ServiceId = "HelloWorldApp";
+                })
+                .ConfigureLogging(logging => logging.AddConsole())
+                .Build();
+
+            await client.Connect(RetryFilter);
+            Console.WriteLine("Client successfully connect to silo host");
+            return client;
+        }
+
+```
+
+| Option | Used for |
+|-------------|----------|
+| `.UseLocalhostClustering()` | Same as for SiloHost |
+| `ClusterOptions` | Same as for SiloHost |
+
+A more in-depth guide to configuring your client can be found [in the Client Configuration section of the Configuration Guide.](http://dotnet.github.io/orleans/Documentation/clusters_and_clients/configuration_guide/client_configuration.html)
+
+## Writing a grain
+
+Grains are the key primitives of the Orleans programming model.
+Grains are the building blocks of an Orleans application, they are atomic units of isolation, distribution, and persistence.
+Grains are objects that represent application entities.
+Just like in the classic Object Oriented Programming, a grain encapsulates state of an entity and encodes its behavior in the code logic.
+Grains can hold references to each other and interact by invoking each other’s methods exposed via interfaces.
+
+You can read more about them in the [Core Concepts section of the Orleans documentation.](http://dotnet.github.io/orleans/Documentation/core_concepts/index.html)
+
+This is the main body of code for the Hello World grain:
+
+```csharp
+[...]
+namespace HelloWorld.Grains
+{
+    public class HelloGrain : Orleans.Grain, IHello
+    {
+        Task<string> IHello.SayHello(string greeting)
+        {
+            logger.LogInformation($"SayHello message received: greeting = '{greeting}'");
+            return Task.FromResult($"You said: '{greeting}', I say: Hello!");
+        }
+    }
+}
+```
+
+A grain class implements one or more grain interfaces, as you can read [here, in the Grains section.](http://dotnet.github.io/orleans/Documentation/grains/index.html))
+
+```csharp
+[...]
+namespace HelloWorld.Interfaces
+{
+    public interface IHello : Orleans.IGrainWithIntegerKey
+    {
+        Task<string> SayHello(string greeting);
+    }
+}
+
+```
+
+## How the parts work together
+
+This programming model is built as part of our core concept of distributed Object Oriented Programming.
+SiloHost is run first.
```

Comment:
is run? is this grammar correct? 
should it be "is started" ?

Reply:
It is correct, but it looks awkward. I'll put "is started" back.

Reply:
I did the same to the next line about the client, too.